src/Primary_Extension/org/lobobrowser/primary/ext/ComponentSource.java

```diff
@@ -73,7 +76,11 @@
   private final JButton searchButton;
   private final JButton reqManagerButton;
   private final ActionPool actionPool;
-  private final DirectorySource directorySource;
+  private final DirectorySource directorySource;  
+  
+  //Key Stroke for CMD in case of MAC and CTRL otherwise
+  public static final int CMD_CTRL_KS = PlatformInit.OS_NAME == OS.MAC ? Toolkit.getDefaultToolkit().getMenuShortcutKeyMask()
```

Comment:
Can you rename this to `CMD_CTRL_KEY_MASK` and update the comment to say "mask for keystroke"?